src/main/resources/assets/admin/common/js/ui/time/DateTimePicker.ts

```diff
@@ -72,6 +79,15 @@ export class DateTimePicker
         if (builder.date) {
             this.setDate(builder.date);
         }
+
+        if (builder.defaultDate) {
+            this.setDefaultHandler(
+                () => {
+                    this.setSelectedDateTime(builder.defaultDate);
+                },
+                builder.defaultDate
+            );
+        }
```

Comment:
See comment for DatePicker.

A side note: we should refactor DateTimePicker to extend/inherit from DatePicker to eliminate code duplication. Same with their builders.